pubsub/redis/redis.go

```diff
@@ -37,7 +57,13 @@ func NewRedisStreams(logger logger.Logger) pubsub.PubSub {
 }
 
 func parseRedisMetadata(meta pubsub.Metadata) (metadata, error) {
-	m := metadata{}
+	// Default values
+	m := metadata{
+		processingTimeout: 5 * time.Second,
```

Comment:
Create a docs issue in `dapr/docs` to update docs for the new metadata and default values.

Reply:
Created https://github.com/dapr/docs/issues/1255